Source/dafny.msbuild/build/dafny.msbuild.targets

```diff
@@ -14,10 +14,15 @@
             Outputs="$(IntermediateOutputPath)GeneratedFromDafny.cs">
         <PropertyGroup>
             <DafnyOutputFile>$(IntermediateOutputPath)GeneratedFromDafny.cs</DafnyOutputFile>
+			<AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
         </PropertyGroup>
         <Message Text="Compiling Dafny source files to $(DafnyOutputFile)..." Importance="high"/>
-        <Exec Command="$(DafnyBinariesDir)$(Dafny) /out:$(DafnyOutputFile) @(DafnySource, ' ') /compile:0 /spillTargetCode:3 /noVerify" />
```

Comment:
You should just be able to add `... @(CompileDafnyPassthrough, ' ') ...` to this line somewhere.